core/src/main/java/org/apache/accumulo/core/client/PluginEnvironment.java

```diff
@@ -122,6 +121,16 @@
      */
     @Override
     Iterator<Entry<String,String>> iterator();
+
+    /**
+     * Returns a derived value from this Configuration. The returned value supplier is thread-safe
+     * and attempts to avoid re-computation of the response. The intended use for a derived value is
+     * to ensure that configuration changes that may be made in Zookeeper, for example, are always
+     * reflected in the returned value.
+     *
+     * @since 2.1.0
```

Comment:
This actually doesn't need the since tag, since the entire class is new in 2.1.

Reply:
Thanks, updated for the next commit.